slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -478,6 +487,10 @@ class SlideToActView @JvmOverloads constructor (
                 }
                 MotionEvent.ACTION_MOVE -> {
                     if (mFlagMoving) {
+
+                        //True if the cursor was not at the end position before this event
+                        val wasIncomplete = mPositionPerc < 1f
+
```

Comment:
?

Reply:
A flag that tells if the cursor wasn't at the end of it's path before the event got called.
Not checking this will cause repeated calls to handleVibration() till the user holds his finger on the cursor.